.github/workflows/pytest.yml

```
@@ -10,24 +10,28 @@ jobs:
         pip install pipenv codecov
         pip install -r requirements.txt
         pipenv install --dev
+
     - name: Test with pytest
       run: |
         sudo mkdir /var/log/s3_upload
         sudo chmod o+w /var/log/s3_upload
         pytest -v --cov --count 10 --random-order tests/unit/
+
     - name: Get test coverage % from pytest output for coverage badge
       run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:26: Double quote to prevent globbing and word splitting

(shellcheck)